SQL/0000-00-00-schema.sql

```diff
@@ -495,6 +495,7 @@ CREATE TABLE `files` (
   `Caveat` tinyint(1) default NULL,
   `TarchiveSource` int(11) default NULL,
   `ScannerID` int(10) unsigned default NULL,
+  `AcqOrderPerModality` int(11) default NULL,
```

Comment:
so @driusan 
changes were done on the LORIS-MRi side to refer to this column name here.